recipes/eleuther_eval.py

```diff
@@ -57,13 +317,16 @@ def __init__(
         max_seq_length: int = 4096,
         batch_size: int = 8,
         dtype: torch.dtype = torch.float32,
+        enable_kv_cache: bool = True,
     ):
+        # TODO (@joecummings): Remove this init function so we don't load in extraneous stuff
```

Comment:
Sorry what does this mean? Are some of these fields unused?

Reply:
Uhmmmmm, turns out we actually load in a copy of a GPT-2 model (not that large so es okay) when we do this call. We overwrite anything that would affect generation, but it's extra memory overhead we shouldn't need.

Reply:
How big? If nontrivial let's def file a follow-up task (I assume we can just load in some dummy empty model instead?)